glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 283f982821510d6c8c5e48054919974b2b2b9664b838ec9f9a06705e19ed3d99
```

Comment:
Not sure.  I did do a real glide update, so I would have expected the hash to change, but I guess not?